content/en/getting_started/integrations/aws.md

```diff
@@ -0,0 +1,300 @@
+| [VPC (Virtual Private Cloud)][67]       | Launch AWS resources into a virtual network                                            |
+| [Web Application Firewall (WAF)][68]    | Protect web applications from common web exploits                                      |
+| [Workspaces][69]                        | Secure desktop computing service                                                       |
+| [X-Ray][70]                             | Tracing for distributed applications                                                   |
+
+## Sending Logs
+
+As a result of setting up the Datadog Forwarder function with the CloudFormation template, you can [automatically set up triggers][19] to collect AWS logs from multiple sources in CloudWatch and S3. Click into the `Collect Logs` tab in the Integration tile and enter the ARN of the Forwarder Lambda. With the ARN entered, click the checkmark to establish the connection:
+{{< img src="getting_started/integrations/integration-tile-collect-logs-1.png" alt="An image displaying the 'Collect Logs' tab of the AWS integration tile within the Datadog account. Along the left is a section displayed as 'AWS accounts' which shows the obfuscated AccountID of the connected account. On the right is a section displayed as 'Enter a Lambda ARN' which shows the ARN of the Datadog Forwarder Lambda function, with the accountID section of the ARN obfuscated.">}}
+
+Once this is successful, you can select from any of the log sources provided in the list. For this guide only S3 access logs have been selected. Click `Save`:
+{{< img src="getting_started/integrations/integration-tile-collect-logs-2.png" alt="An image displaying the 'Collect Logs' tab of the AWS integration tile within the Datadog account. Along the left is a section displayed as 'AWS accounts' which shows the obfuscated AccountID of the connected account. On the right is a section displayed as 'Currently used Lambda ARNs' which shows the ARN of the Datadog Forwarder Lambda function, with the accountID section of the ARN obfuscated. Under this are the options for 'Enter a Lambda ARN' and 'Select the services you would like to collect logs for'. The only option which is ticked is the 'S3 Access Logs' option.">}}
+
+If needed, check out [Enable logging for your AWS service][63] to start generating logs on your AWS account. Once you're receiving logs, review [Log Management][64] for more information about processing, enriching, searching, and analyzing your logs.
+
+### Validation
+
+Find your logs in the [Logs Explorer][20] using either the `source` or `service` facets from the facet panel:
+{{< img src="getting_started/integrations/logs-explorer.png" alt="An image from the Logs Explorer page of the Datadog account. Along the left the image displays the Source and Service facets, both checked with 's3'. Along the right, some log entries are displayed in a list format.">}}
+
+## Deeper visibility with the Datadog Agent on a host
```

Comment:
ECS is also container, so  I would move to the containers section.

Would relabel to `Deeper visibility with the Datadog Agent on EC2`